examples/jsm/utils/GeometryCompressionUtils.d.ts

```diff
@@ -0,0 +1,9 @@
+import * as THREE from "../../../build/three.module.js";
+
+export namespace GeometryCompressionUtils {
+
+	export function compressNormals( mesh: THREE.Mesh, encodeMethod: String );
```

Comment:
These functions should have `: void` return values.

Reply:
Thank you 👍 , will improve the codes in the next PR.